lib/spack/spack/test/conftest.py

```diff
@@ -2072,6 +2072,11 @@ def pytest_runtest_setup(item):
     if only_windows_marker and sys.platform != "win32":
         pytest.skip(*only_windows_marker.args)
 
+    # XFail test marked "requires_builtin" if builtin repo is required
+    requires_builtin_marker = item.get_closest_marker(name="requires_builtin")
+    if requires_builtin_marker and not os.path.exists(spack.paths.packages_path):
+        pytest.xfail(*requires_builtin_marker.args)
```

Comment:
Should we `xfail` (run the test and expect failure) or `skip`?

Reply:
I debated that too. My recollection was that Todd said in the meeting to XFAIL in case we want to fix that at some point. But I could go either way.

Reply:
In the interest of time, I'll switch to skipping.